var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -41,6 +41,17 @@ class NetcdfCxx4(AutotoolsPackage):
 
     force_autoreconf = True
 
+    def configure_args(self):
+        args = []
+
+        CPPFLAGS = []
```

Comment:
I would use lowercase letters for the variable name. PEP 8 says to use all caps for global variables and lowercase for local variables.

Reply:
In this case, the "Python" variable actually mirrors an environment variable of the same name.  Naming the two the same is quite natural, regardless of what PEP8 says about "normal" Python variables.  Naming the two different introduces unnecessary complexity just to conform to PEP8.

As Winston Churchill is reputed to have said: “This is the sort of English up with which I will not put.”   Can we bend on this one?